Buổi 11/homework.js

```diff
@@ -0,0 +1,115 @@
+//Bài 1
+const addElement =( a, number ) => {
+  const arr=[]
+  for(let i=0; i<number; i++){
+    arr.push(a)
+  }
+  return arr
+}
+
+//Bài 2
+const sortArr = ( a = [] ) => {
+  const newArr = []
+    for(let i=a.length-1; i >= 0; i--){
+        newArr.push(a[i])
+    }
+  return newArr
+}
+
+//Bài 3
+const check = ( arr = [] ) => {

```

Comment:
`check` là check cái gì đây?
```suggestion
const check = (arr = []) => {
```